data/de.haeckerfelix.gradio.gschema.xml

```diff
@@ -30,6 +30,11 @@
       <summary>Show notifications</summary>
     </key>
 
+    <key name="show-technical-information" type="b">
+      <default>true</default>
```

Comment:
I think it's better to turn it off by default. Normal users should not need this information.

Reply:
okay. I have a feeling that most users want to go for the highest quality, and that it's going to be too hidden now.